src/Microsoft.Azure.WebJobs.Extensions.OpenApi.CLI/Service/Interface/ICustomOpenApiCreator.cs

```diff
@@ -0,0 +1,17 @@
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Enums;
+using Microsoft.OpenApi.Models;
+
+namespace Microsoft.Azure.WebJobs.Extensions.OpenApi.CLI.Service.Interface
```

Comment:
Can we change the folder structure from `Service.Interface` to `Abstractions` and also change the namespace accordingly?

Reply:
Changed to match request in new commit